instrumentation/active_job/test/instrumentation/active_job/patches/active_job_callbacks_test.rb

```diff
@@ -205,20 +204,14 @@
       end
 
       it 'tracks correctly for jobs that do retry' do
-        ::ActiveJob::Base.queue_adapter = :async
-
-        RetryJob.perform_now
-        ::ActiveJob::Base.queue_adapter.shutdown
-
-        # 1 enqueue, 2 perform
-        _(spans.count).must_equal(3)
-
-        span = spans.last
-        _(span.kind).must_equal(:consumer)
-        _(span.attributes['messaging.active_job.executions']).must_equal(2)
+        begin
+          RetryJob.perform_later
+        rescue StandardError
+          nil
+        end
 
-      ensure
-        ::ActiveJob::Base.queue_adapter = :inline
+        executions = spans.filter { |s| s.kind == :consumer }.map { |s| s.attributes['messaging.active_job.executions'] }.sum
```

Comment:
An absolutely minor nit for your consideration:

```suggestion
        executions = spans.filter { |s| s.kind == :consumer }.sum { |s| s.attributes['messaging.active_job.executions'] }
```

```ruby
[1] pry(main)> a = [ { foo: 1 }, { foo: 2 }, { foo: 3 } ]
=> [{:foo=>1}, {:foo=>2}, {:foo=>3}]
[2] pry(main)> a.sum { |s| s[:foo] }
=> 6
[3] pry(main)>
```